package.json

```diff
@@ -16,14 +16,15 @@
     "@mdi/font": "^5.0.45",
     "@sentry/browser": "^5.15.4",
     "@sentry/integrations": "^5.15.4",
+    "@vysakh-aot/detail-comment": "^1.0.4",
     "array.prototype.move": "0.0.4",
     "axios": "^0.19.2",
     "core-js": "^3.6.4",
     "http-status-codes": "^1.4.0",
     "launchdarkly-js-client-sdk": "^2.17.0",
     "lodash": "^4.17.15",
     "regenerator-runtime": "^0.13.3",
-    "sbc-common-components": "2.3.3",
+    "sbc-common-components": "2.3.4",
```

Comment:
I think there is even a 2.3.5 by now :P

Reply:
Oh, there is. Geez, I just did this yesterday!
I think getting the latest update can wait... I don't know what's in it anyway.

Reply:
We should update all 3 UI projects to the latest sbc-common-components... after we learn what's in it... I will create a new ticket for that.

Reply:
I think this latest update was changes to `sbc-authentication-options`, it was made to fix a redirect issue happening in name request. Nothing affecting this UI for sure 👍